### ISSUE TITLE
Translation Platform

### ISSUE THREAD

**Issue by jordisala1991** [sonata-project/dev-kit#1614]
Continues : #98 Seems like we already talked a bit about it

**Current status:**
- Handling translations is hard, and it is harder to have a lot of languages we don't even understand.
- Translators are harder to get too, and we get most contributions from developers that speak the language and wants sonata to be translated in his/her language.
- We end up with translation files that are outdated or missing keys.
- We can't track the translation status for the whole project or for a single language

**Desired situation:**
- We would like to track the status of the different translations
- We would like to get proffesional translators
- We would like to have a nice way to edit translations which does not involve having developer skills

**How do we do it?**
We integrate a third party platform, in this case my proposal is https://weblate.org/en/
We have access to a platform that is open source, and have plans for open source projects.
With a nice interface that not only allows translators to do the work with more information but it also catches some mistakes or missing keys, or changed source translations, etc...

**Roadmap**
- [ ] Test with SonataAdminBundle and SonataMediaBundle
- [ ] Modify editorconfig and linter for xml on all projects
- [ ] Modify readmes and contribution guides
- [ ] Activate Weblate on all projects
- [ ] Tweet or something to let people know

**Comment by VincentLanglet** [comment]
> * [ ]  Modify editorconfig and linter for xml on all projects

https://github.com/sonata-project/dev-kit/pull/1615

**Comment by VincentLanglet** [comment]
Will also need to modify all the `origin="AdminFooBundle.en.xliff"`

**Comment by VincentLanglet** [comment]
I fixed xliff files in
- SonataDatagridBundle
- SonataDashboardBundle
- SonataCommentBundle
- SonataAdminBundle
- twig-extensions
- SonataTranslationBundle
- SonataTimelineBundle

**Comment by jordisala1991** [comment]
This is the full list.
Before anything, we should remove the placeholders translations and leave only the good ones for each project. I leave the list here to keep updating it.

Once that step is done, we should check for the SonataAdmin, to see if they can be removed, and then do a list with all the context that need to have each project. For example, there are project with normal translations and validations (that will require dou

Also would be nice to have the dev-kit pr to get the `cs-fix-xliff` @VincentLanglet 

**Removed placeholders translations**
- [ ] SonataDatagridBundle
- [ ] SonataDashboardBundle
- [ ] SonataCommentBundle
- [ ] SonataAdminBundle
- [ ] twig-extensions
- [ ] SonataTranslationBundle
- [ ] SonataTimelineBundle
- [ ] SonataUserBundle
- [ ] SonataSeoBundle
- [ ] SonataPageBundle
- [ ] SonataNotificationBundle
- [ ] SonataNewsBundle
- [ ] SonataMediaBundle
- [ ] form-extensions
- [ ] SonataFormatterBundle
- [ ] ecommerce
- [ ] SonataClassificationMediaBundle
- [ ] SonataClassificationBundle
- [ ] SonataArticleBundle
- [ ] SonataBlockBundle
- [ ] SonataPHPCRAdminBundle

**Check for all contexts**
- [ ] SonataDatagridBundle
- [ ] SonataDashboardBundle
- [ ] SonataCommentBundle
- [ ] SonataAdminBundle
- [ ] twig-extensions
- [ ] SonataTranslationBundle
- [ ] SonataTimelineBundle
- [ ] SonataUserBundle
- [ ] SonataSeoBundle
- [ ] SonataPageBundle
- [ ] SonataNotificationBundle
- [ ] SonataNewsBundle
- [ ] SonataMediaBundle
- [ ] form-extensions
- [ ] SonataFormatterBundle
- [ ] ecommerce
- [ ] SonataClassificationMediaBundle
- [ ] SonataClassificationBundle
- [ ] SonataArticleBundle
- [ ] SonataBlockBundle
- [ ] SonataPHPCRAdminBundle

**Configure Weblate**
- [ ] SonataDatagridBundle
- [ ] SonataDashboardBundle
- [ ] SonataCommentBundle
- [ ] SonataAdminBundle
- [ ] twig-extensions
- [ ] SonataTranslationBundle
- [ ] SonataTimelineBundle
- [ ] SonataUserBundle
- [ ] SonataSeoBundle
- [ ] SonataPageBundle
- [ ] SonataNotificationBundle
- [ ] SonataNewsBundle
- [ ] SonataMediaBundle
- [ ] form-extensions
- [ ] SonataFormatterBundle
- [ ] ecommerce
- [ ] SonataClassificationMediaBundle
- [ ] SonataClassificationBundle
- [ ] SonataArticleBundle
- [ ] SonataBlockBundle
- [ ] SonataPHPCRAdminBundle

**Comment by jordisala1991** [comment]
First example on how to do a PR to remove placeholders: https://github.com/sonata-project/SonataMediaBundle/pull/2088

We need to do all bundles, a little of work if anyone wants to help @sonata-project/contributors
